examples/simple_counter/.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[alias]
+uniffi-gen-swift = "run --bin uniffi-bindgen generate --library target/release/libshared.dylib --language swift --out-dir shared/generated/swift/SharedFFI"
+uniffi-gen-java = "run --bin uniffi-bindgen generate --library target/release/libshared.dylib --language kotlin --out-dir shared/generated/java"
```

Comment:
Do we need an equivalent for the wasm version? Mostly a question for @StuartHarris I suppose, I don't 100% remember how that stage of the build process works.

Reply:
I'm not convinced either of these 2 lines is relevant now, as there is no `.udl` file anymore